frontend/src/components/form/Input.jsx

```diff
@@ -22,6 +26,43 @@ export const TextArea = styled(AntTextArea)`
     border: white;
   }
 `;
+export const NumericInput = styled(AntNumericInput)`
+  &&& {
+    border-radius: 8px;
+    padding: 0;
+    background-color: ${props => props.backgroundColor};
+    border: white;
+    max-width: 5rem;
+
+    input {
+      height: 2.5rem;
+    }
+  }
+`;
+
+const SelectWrapper = styled(AntSelect)`
+  && {
+    background-color: ${props => props.backgroundColor};
+    border: white;
+    width: 100%;
+
+    div {
+      border-radius: 8px;
+      width: 100%;
+    }
+  }
+`;
+
+export const Select = ({ data, ...rest }) => (
```

Comment:
As stated before, try to avoid taking "rest" of the props in a variable, but instead explicitly take the ones you need.